src/Makefile.in

```diff
@@ -184,7 +184,7 @@ endif
 #  also need to figure out -mpreferred-stack-boundary=4 and add (or whatever the stack size is)
 #
 #########################################################
-default:
+default: unit-tests
 	$(MAKE) find_version
```

Comment:
I am going to fail this, which you can override, due to the unit test being done (AND RUN) at each make.  Doing the build would be acceptable, if we created a 'separate' target for unit-tests-run:   or something like that.

ALSO, it should not be default: unit-tests, but
```
default:
	$(MAKE) unit-tests
	$(MAKE) find_version
	...
```

I 'think'